src/components/editor/diacritics/CharacterButton.jsx

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useSelector } from 'react-redux'
+
+const CharacterButton = (props) => {
+  const targetRef = useSelector(state => state.selectorReducer.editor.diacritics.targetRef)
+
+  const handleClick = (event) => {
+    const existingValue = targetRef.current.value
+    targetRef.current.value = existingValue + props.character
+    targetRef.current.focus()
+    event.preventDefault()
+  }
+
+  return (
+    <button className="btn btn-light"
+            style={{ margin: '.25em' }}
```

Comment:
Most of our other margins are in pixels. Can you switch?

Reply:
Done.